Exercise6/Server/index.js

```diff
@@ -0,0 +1,38 @@
+const data = [
+  {
+    name: "Neeraj Yadav",
+    company: "Infuse",
+    role: "Software Developer In Test",
+  },
+  {
+    name: "Madhura Patil",
+    company: "Infuse",
+    role: "Software Developer In Test",
+  },
+  {
+    name: "Riya Kumbhar",
+    company: "Infuse",
+    role: "Software Developer In Test",
+  },
+  {
+    name: "Vaibhav Gaunkar",
+    company: "Infuse",
+    role: "Software Developer In Test",
+  },
+];
+
+const express = require("express");
+const app = express();
+const PORT = 8000;
+const CORS = require("cors");
```

Comment:
Move the require line for cors after the express require line. This maintains a cleaner structure by following the order of first importing external dependencies and then defining our own constants and configurations.